agenta-web/cypress/e2e/regex-evaluation.cy.ts

```diff
@@ -0,0 +1,229 @@
+import {randString} from "../../src/lib/helpers/utils"
+
+describe("Regex Evaluation workflow", () => {
+    let app_id
+    context("When creating variants and testsets", () => {
+        it("Should create app variant", () => {
```

Comment:
can we put this test in a separate file so we can import it in different tests ?
we will need to create same test in many other features